src/AdminConsole/Pages/Organization/Admins.cshtml.cs

```diff
@@ -3,13 +3,15 @@
 using Microsoft.AspNetCore.Identity;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.AspNetCore.RateLimiting;
 using Passwordless.AdminConsole.EventLog.Loggers;
 using Passwordless.AdminConsole.Helpers;
 using Passwordless.AdminConsole.Identity;
 using Passwordless.AdminConsole.Services;
 
 namespace Passwordless.AdminConsole.Pages.Organization;
 
+[EnableRateLimiting("fixed")]
```

Comment:
Remove the rate limiting on page level, instead move OnPostInvite to an endpoint, and have the form submission calling the endpoint.

Then apply the rate limiting to the endpoint.

`app.MapGet("/", () => Results.Ok($"Fixed Window Limiter {GetTicks()}"))
                           .RequireRateLimiting(fixedPolicy);`
                           
Consider also using a static readonly representation of the rate limiting policy.

Reply:
Chatted with @abergs about splitting this out into a separate API endpoint. He felt it wasn't worth moving it out of the page.  We decided upping the limit to 100 and put a few more safe guards in.  Only allowing 1 invite per email at a time and 10 pending invites at a time.

Also went ahead with the static definition of the policy and adding of rate limiting.

Reply:
Can you not use something better, i.e.:

1. Preventing double submissions in the first place
2. Upon generating links, I suppose they are valid for a limited time, so you can only have up to 3 active magic links for a given user or admin.